Gruntfile.js

```diff
@@ -953,10 +958,14 @@ module.exports = function(grunt) {
     'exec:pack-node-modules',
   ]);
 
-  grunt.registerTask('start-webdriver', 'Starts Protractor Webdriver', [
-    'replace:webdriver-version',
-    'exec:start-webdriver',
-  ]);
+  grunt.registerTask('start-webdriver', 'Starts Protractor Webdriver', function() {
```

Comment:
Can we use the format we've been using for the rest of the grunt tasks: 
```
grunt.registerTask('start-webdriver', 'Starts Protractor Webdriver', [
  'replace:webdriver-version',
  CI ? 'exec:start-webdriver-ci' : 'exec:start-webdriver',
]);

Reply:
switched!